File: app/src/main/java/can/siempredelao/f1kotlin/backend/MRDataStandings.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class MRDataStandings {

    public String         xmlns;
    public String         series;
    public String         url;
    public String         limit;
    public String         offset;
    public String         total;
    @SerializedName("StandingsTable")
    public StandingsTable standingsTable;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/MRDataStandings.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class MRDataStandings(val xmlns: String,
                      val series: String,
                      val url: String,
                      val limit: String,
                      val offset: String,
                      val total: String,
                      @SerializedName("StandingsTable") val standingsTable: StandingsTable)
